modules/persistent-storage-csi-cloning-provisioning.adoc

```diff
@@ -5,22 +5,20 @@
 [id="persistent-storage-csi-cloning-provisioning_{context}"]
 = Provisioning a CSI volume clone
 
-Provisioning of a CSI volume clone is triggered by the creation of a cloned persistent volume claim (PVC) API object. The cloned PVC is pre-populated with the contents of another PVC. After creation, the cloned volume adheres to the same rules as any other persistent volume.
+When you create a cloned persistent volume claim (PVC) API object, you trigger the provisioning of a CSI volume clone. The clone pre-populates with the contents of another PVC, adhering to the same rules as any other persistent volume.
+
+The one exception of cloned provisioning is that you must add a `dataSource` that references an existing PVC in the same namespace.
 
 .Prerequisites
-* Logged in to a running {product-title} cluster.
-* A PVC created using a CSI driver that supports volume cloning.
-* Storage back end is configured for dynamic provisioning. Cloning support is not available for static provisioners.
-* Source and destination PVCs must exist in the same namespace.
-* Source and destination storage class must be the same.
+* You are logged in to a running {product-title} cluster.
+* Your PVC is created using a CSI driver that supports volume cloning.
+* Your storage back-end is configured for dynamic provisioning. Cloning support is not available for static provisioners.
```

Comment:
back end is two words when it's a noun.

Reply:
Thanks, I forgot why I had it two different ways. I hope they change their guidance to "backend" (always one word), as that's how I most often see it.